src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+/* stylelint-disable string-quotes */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable color-hex-case */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  font-family: 'Roboto';
+  padding: 0 50px;
+  align-items: center;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__link:hover {
```

Comment:
Please arrange all your selectors in the order they are in html:
.header { ... }
.logo { ... }
.nav { ... }
.nav__list { ... }
.nav__list-item { ... }
.is-active { ... }